Core now requiring react - no longer framework agnostic?

https://github.com/Typeform/embed/blob/main/packages/embed/src/base/iframe-options.ts#L1

This line breaks existing vuejs projects cos it now requires that react is installed.

Any help on how to fix this? Thanks in advance.

Ref error message for others who might be searching:
```
Cannot find module 'react' or its corresponding type declarations.
```

--- a/packages/embed/src/base/iframe-options.ts
+++ b/packages/embed/src/base/iframe-options.ts
@@ -1,5 +1,3 @@
-import { IframeHTMLAttributes } from 'react'
-
 export type IframeOptions = {
-  iframeProps?: IframeHTMLAttributes<HTMLIFrameElement>
+  iframeProps?: Partial<HTMLIFrameElement>
 }

